docs/source/user/fast.farm/ModelGuidance.rst

```diff
@@ -812,6 +818,13 @@ parameters:
    .. math::
       \textbf{dr} \le c_\text{max}
 
+When **Wake_Mod=2,3**, for numerical stability, it is recommended to set the spacing with a value that (approximately) satisfies the following guideline (see Equation 20 of the following `paper <https://doi.org/10.5194/wes-6-555-2021>`__):
+
+   .. math::
+      \textbf{dr}  \ltrapprox \frac{D}{20}
```

Comment:
I see dy = D/10 in the paper.  Wouldn't this be equivalent to dr < D/10?

Reply:
yes @jjonkman , you are correct, please change to:

`\textbf{dr}  \ltrapprox \frac{D}{10}`